examples/ads.amp.html

```diff
@@ -1130,6 +1130,14 @@ <h2>Innity</h2>
       data-channel="NEWS">
```

Comment:
Seems like you forgot to add yourself to the example dropdown for all of the vendors 😄 

See: https://github.com/ampproject/amphtml/pull/20367/files

Reply:
I have added Jubna in all required pages now.

Reply:
https://github.com/ampproject/amphtml/pull/20109/files

Reply:
P.S This is the original comment I made asking about the dropdown 😄